service/history/workflow/update/update.go

```diff
@@ -268,20 +264,18 @@ func (u *Update) WaitLifecycleStage(
 	return statusAdmitted(), nil
 }
 
-// abortWaiters fails update futures with abortWaiterErr error and set state to stateAborted.
-func (u *Update) abortWaiters() {
+// abortWaiters fails update futures with reason.Error() error and set state to stateAborted.
+func (u *Update) abortWaiters(reason AbortWaiterReason) {
```

Comment:
I keep seeing the comments say "abort waiters and set state to stateAborted". It's a bit odd that the function is named after half of what it does. Shouldn't we just call it `abort` instead and have the reader checkout what "abort" means by visiting the function's documentation and then learning that it does two things - rather than mislead them with this overly specific name?

Reply:
I agree. I renamed `abortWaiters` to `abort` and updated doc string of the function.